### DISCUSSION TITLE
iOS App Store scam

### DISCUSSION THREAD

**Discussion by ciprain** [WalletConnect/walletconnect-monorepo#955] [Developer Support]
Hi guys. Just wanted to let you know that there’s an app on AppStore called walletconnect which I downloaded few days ago and because I was just trying to connect my trust wallet to http://highpoint.finance without knowing that you could do this through the actually app(still learning 😞), went on this scam app and put my secret phrase there and in like 7 minutes they took all my coins from my trust wallet(aprox 1300$). Please please check this because I don’t want others to go through this. I have already spoken with Apple as well but it looks like they don’t really care 😩. I am waiting for some knowledgeable opinions

**Comment by chadyj** [comment]
I'm sorry to hear you got scammed!

Even before you posted we submitted a scam report to Apple here https://reportaproblem.apple.com/
But almost a week later the app is still available.

As you found out, there is no WalletConnect app.  Instead, WalletConnect is integrated in hundreds of other apps, which can be found here https://walletconnect.com/registry.

We also encourage everyone to get familiar with these safety tips https://walletconnect.com/faq#safety
